enterprise/server/quota/quota_manager.go

```diff
@@ -362,7 +373,8 @@ func Register(env environment.Env) error {
 	if !*quotaManagerEnabled {
 		return nil
 	}
-	qm, err := NewQuotaManager(env)
+	ps := pubsub.NewPubSub(env.GetDefaultRedisClient())
```

Comment:
Any reason you pass the pubsub and env into NewQM() instead of just env and creating the pubsub inside?